cdbcli/environment.py

```diff
@@ -1,20 +1,63 @@
+import contextlib
+import io
+import subprocess
 import sys
 
 
-class Environment(object):
+class Environment():
```

Comment:
Oh my!  You're right [judging by the tutorial examples](https://docs.python.org/3/tutorial/classes.html#class-objects)...